cmd/release-controller/jira.go

```diff
@@ -101,6 +172,28 @@ func (c *Controller) syncJira(key queueKey) error {
 			break
 		}
 	}
+
+	// Auto-populate verifyIssues for -nightly releases if not already set
+	if verifyIssues == nil && strings.HasSuffix(release.Config.Name, "-nightly") {
```

Comment:
This logic never matches:
`"name": "4.21.0-0.nightly",`
From: https://github.com/openshift/release/blob/2997620fbf426c4e644bf3bd2a74e07844715f2b/core-services/release-controller/_releases/release-ocp-4.21.json#L2

Reply:
Thanks. Updated.